.bluemix/deploy.json

```diff
@@ -87,6 +91,16 @@
                   "type": "select",
                   "key": "selected-space",
                   "readonly": false
+                },
+                {
+                  "type": "label",
+                  "title": "To deploy IBM Cloud Functions and Cloud Foundry applications, the pipelines require an [IBM Cloud Platform API key](https://console.bluemix.net/docs/cli/reference/bluemix_cli/bx_cli.html#ibmcloud_iam_api_key_create). Use the `ibmcloud iam api-key-create <name>` command to create a new API key if you do not already have a key."
```

Comment:
we have cloud functions in cloudco??

Reply:
layout is a bit weird:
![image](https://user-images.githubusercontent.com/12746550/42322748-d96483a8-805d-11e8-9d2e-6258913fc848.png)

in LW, it uses the full width:
![image](https://user-images.githubusercontent.com/12746550/42322770-ef011cb2-805d-11e8-99a3-d206c026b890.png)